src/5.0_to_5.1/migrateFilters.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Wraps the MDX of each filter in `filters`.
```

Comment:
```suggestion
 * Wraps the MDX of each filter in `mdx`.
```
Right?

Reply:
Hmm no that's what I meant: each element of `filters` is a filter.
In 5.0 it is represented by raw mdx, and in 5.0 it must have the form `{mdx}`

Reply:
Aaah, okay, I misunderstood the sentence. I find the comment confusing, but I don't have a clear idea of a good improvement for it. Maybe this?
```suggestion
 * Wraps each filter in `filters` in an object.
```

Also I don't see the need for "the MDX of", each 5.0 filter is an MDX